e2e/e2e_suite_test.go

```diff
@@ -42,19 +43,28 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 			fmt.Printf("\nExposing Docker Registry\n")
 			<-cmd.Execute("oc", "expose", "service", "docker-registry", "-n", "default").Done()
 
-			// create a 'real user' we can use to push to the DockerRegsitry
+			// create a 'real user' we can use to push to the DockerRegistry
 			fmt.Printf("\nAdd admin user\n")
 			<-cmd.Execute("oc", "create", "user", "admin").Done()
 			<-cmd.Execute("oc", "adm", "policy", "add-cluster-role-to-user", "cluster-admin", "admin").Done()
```

Comment:
Any reason to keep these in code?

Reply:
I thought it's specific to our tests, so whoever is using `Makefile` can use any user / privileges they want